### PR TITLE
Add examples for doctest to optuna/samplers/*.py and optuna/integrati…

### PR DESCRIPTION
**Pull request by nuka137**
This PR add doctest example to below files.

* optuna/samplers/random.py
* optuna/samplers/tpe/sampler.py
* optuna/integration/skopt.py

I changed `n_trials` to `10` in some tests, because `n_trials=100` is too much for testing.  
Also, I fixed an incorrect argument of `study.optimize` about `RandomSampler` doctest.

Related Issue: #734

### CODE REVIEW

File: optuna/integration/skopt.py [hunk 1]
```diff
@@ -46,7 +48,7 @@ def objective(trial):
 
                 sampler = optuna.integration.SkoptSampler()
```
Comment (toshihikoyanase) [inline optuna/integration/skopt.py]: The `SkoptSampler` requires `scikit-optimize`. Please add it to the requirements of `doctest` in `setup.py`.

File: optuna/samplers/random.py [hunk 2]
```diff
@@ -22,10 +22,17 @@ class RandomSampler(BaseSampler):
...
+        .. testcode::
 
-            >>> study = optuna.create_study(sampler=RandomSampler())
-            >>> study.optimize(objective, direction='minimize')
+            import optuna
+            from optuna.samplers import RandomSampler
+
+            def objective(trial):
+                x = trial.suggest_uniform('x', -5, 5)
+                return x ** 2
```
Comment (toshihikoyanase) [inline optuna/samplers/random.py]: Please remove white spaces around the `**` operator as can be seen in doctests for `SkoptSampler` and `TPESampler`.
```suggestion
                return x**2
```

File: optuna/samplers/random.py [hunk 3]
```diff
@@ -22,10 +22,17 @@ class RandomSampler(BaseSampler):
...
+
+            study = optuna.create_study(sampler=RandomSampler(), direction='minimize')
```
Comment (toshihikoyanase) [inline optuna/samplers/random.py]: IMO, the `direction` argument can be removed to emphasize the `RandomSampler`.
```suggestion
            study = optuna.create_study(sampler=RandomSampler())
```

**Comment by nuka137** [comment]
@toshihikoyanase 

Thanks for your review!
I fixed all issues.

**Pull request merged**